net-irc/atheme-services/atheme-services-7.2.10_p2.ebuild

```diff
@@ -25,7 +25,8 @@ RDEPEND=">=dev-libs/libmowgli-2.1.0:2
 	pcre? ( dev-libs/libpcre )
 	ssl? ( dev-libs/openssl:0= )"
 DEPEND="${RDEPEND}
-	virtual/pkgconfig"
+	virtual/pkgconfig
```

Comment:
So, I cant find git installing serno.h, but weirdly it still fixes the build compilation. 
However could you swap places for dev-vcs/git and virtual/pkgconfig so we sort them alphabetically. Then this is good to go!

Reply:
As I recall, `serno.h` is generated by the atheme build process.  It uses the git commit as the SERial N... NOm... N(o)umber.  Or something.  Hence it's only needed at build time and not run time.

I feel shame for not sorting them alphabetically beforehand :)